### ISSUE TITLE
Taskbar icons do not move left when using custom date/time format

### ISSUE THREAD

**Issue by KayGee53** [martinet101/ElevenClock#690]
Hi all,
Not sure if this issue has been written about before, apologies if it has but I couldn't find anything about it.
When using the following custom date format,
%H:%M:%S
%A, %B %d, %Y
The taskbar icons are overwritten and I can't seem to find any way to fix it.
I am attaching a before and after screenshot and my logfile.
Thanks in advance for any help and also for a brilliant utility to overcome the shortcomings of Microsofts messing up of the clock
![Before](https://user-images.githubusercontent.com/101569149/158177305-7b739fe7-0e88-401e-9a38-52f2cf1ba83d.jpg)
![After](https://user-images.githubusercontent.com/101569149/158177308-c2c9a6dd-934b-401d-9b99-94cf16541609.jpg)
[Log.txt](https://github.com/martinet101/ElevenClock/files/8244990/Log.txt)
!

**Comment by martinet101** [comment]
Yes, that's because the clock does not _replace_ the system clock, but rather it shows over it. Achieving the effect of moving the icons is kind of impossible due to MS restrictions. I'm sorry though :(
